src/basic-languages/python/python.ts

```diff
@@ -287,9 +317,13 @@ export const language = <languages.IMonarchLanguage>{
 			[/\\$/, 'string']
 		],
 		fStringDetail: [
-			[/[:][^}]+/, 'string'],
+			[/:=[^}]+/, 'identifier'], // walrus operator
```

Comment:
Can you give me examples of where these would activate? Why are they followed by 'identifier' and not string?